### PR TITLE
chore(app): update to next 8 and styled-components 4

### PR DESCRIPTION
**Pull request by douglasduteil**
---

It helps solving e2e tests sync issues...

### CODE REVIEW

File: packages/app/pages/_document.js [hunk 1]
```diff
@@ -1,35 +1,51 @@
...
+          div#__next {
+            background:white;
+            height: 100%;
+          }
+          `}</style>
           <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
           <meta name="description" content="" />
           <meta name="author" content="" />
-          <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch,Array.prototype.find,Array.prototype.findIndex,Array.prototype.includes,Object.values" />
+          <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch,Array.prototype.find,Array.prototype.findIndex,Array.prototype.includes" />
```
Comment (revolunet) [inline packages/app/pages/_document.js]: Object.values please

**Comment by revolunet** [comment]
Thanks ! you need to test static export and the user view from IE11 : login, import...

**Comment by douglasduteil** [comment]
Screen_Shot_2018-10-25_at_11 02 15_AM

File: packages/app/pages/_document.js (hunk 1 above)
Comment (douglasduteil) [reply packages/app/pages/_document.js]: sry

**Comment by douglasduteil** [comment]
K static export epic fail...

**Pull request closed without merging**